pkg/controller/account/account_controller.go

```diff
@@ -34,6 +34,7 @@ import (
 var log = logf.Log.WithName("controller_account")
 
 const (
+	awsLimit                = 4500
```

Comment:
This requires all our orgs to support at least `4500` accounts.  What would it take to put this in config somewhere we can tweak in cluster, make this a default value?

Reply:
Right now the quickest way to get this in is to hard code it, we can switch it to a config map and then adjust pipelines etc to support the limits late

Reply:
Hold on CM, we also need to look at support for multiple orgs.  So this would probably be tied to  that configuration somewhere (new CRD, part of account pool, something else? tbd)

Reply:
I agree this should be a separate PR as well.